src/Sema.zig

```diff
@@ -2958,13 +2958,10 @@ fn safetyPanic(sema: *Sema, block: *Scope.Block, src: LazySrcLoc, panic_id: Pani
 }
 
 fn emitBackwardBranch(sema: *Sema, block: *Scope.Block, src: LazySrcLoc) !void {
-    const shared = block.inlining.?.shared;
-    shared.branch_count += 1;
-    if (shared.branch_count > block.branch_quota.*) {
+    sema.branch_count += 1;
```

Comment:
`sema` has `branch_quota` but the previous code here mutating the value on the `Inlining.Shared` is still correct.